content/300-guides/500-other/880-troubleshooting-orm/100-help-articles/800-implicit-to-explicit-conversion.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Converting Implicit many-to-many relation to Explicit many-to-many relation'
+metaTitle: 'Learn how to convert implicit many-to-many relation to explicit many-to-many relation'
+metaDescription: 'Learn how to convert implicit many-to-many relation to explicit many-to-many relation'
+---
+
+## Problem
+
+Users sometimes encounter situations where they need to transition from implicit many-to-many relationships between models to explicit ones. This guide provides a step-by-step walkthrough on how to make that conversion.
+
+## Solution
+
+Many-to-many relationships are an important aspect of relational databases, allowing multiple records in one table to be related to multiple records in another table. Prisma provides two approaches to model many-to-many relationships: [implicit](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) and [explicit](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#explicit-many-to-many-relations). 
+
+In this article, we'll guide you through the process of converting an implicit many-to-many relation to an explicit one in Prisma.
+
+Consider these models with implicit many-to-many:
+
+```prisma
+model User {
+  id        Int       @id @default(autoincrement())
+  name      String
+  posts     Post[]
+}
+
+model Post {
+  id        Int       @id @default(autoincrement())
+  title     String
+  authors   User[]
+}
+```
```

Comment:
I think it would be helpful to actually show the SQL of the underlying relation table here. Fundamentally, this guide needs to teach how the already existing relation table is being surfaced in the Prisma schema.